package.json

```diff
@@ -10,26 +10,25 @@
     "pretest": "npm run compile",
     "lint": "eslint src/*.js test-src/*.js",
     "lintfix": "eslint --fix src/*.js test-src/*.js",
-    "test": "npm run lint && npm run compile && NODE_ENV=test mocha test/*_test.js"
+    "test": "npm run lint && NODE_ENV=test mocha test/*_test.js",
+    "prepublish": "npm run compile"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/elenasolomon/azure-blob-storage.git"
   },
-  "dependencies": {
-    "assume": "^1.2.5",
-    "azure-storage": "^1.0.1",
+  "devDependencies": {
+    "babel-eslint": "^7.0.0",
```

Comment:
This should be safe, but I just wanted to note that this is using a different version of the babel parser than the version that we'll compile against.  This could lead to odd behaviour.  Ideally, we'd get the corresponding version of the babel compiler by looking at babel-compile's dependencies, but this is not important, just an FYI.